action.sh

```
@@ -0,0 +1,8 @@
+#!/system/bin/sh
+MODDIR=/data/adb/modules/SMTW
+function abort() {
+    echo "- $1"
+    sleep 0.5
+    exit 1
+}
```

Shellcheck (0.10.0)

[warning] 'function' keyword is non-standard. Delete it.

(SC2112)